how-to/how-to-use-virtio-mem-with-kata.md

```diff
@@ -0,0 +1,49 @@
+# Kata Containers with `virtio-mem`
+
+- [Introduction](#introduction)
+- [Requisites](#requisites)
+- [Run a Kata Container with utilizing `virtio-mem`](#run-a-kata-container-utilizing-virtio-mem)
+
+## Introduction
+
+The basic idea of `virtio-mem` is to provide a flexible, cross-architecture memory hot plug and hot unplug solution that avoids many limitations imposed by existing technologies, architectures, and interfaces. More details can be found in https://lkml.org/lkml/2019/12/12/681.
```

Comment:
Nit: Very long line. I'd break at 80 chars or similar to make diffs easier to read.

Reply:
done.